src/Microsoft.AspNet.Mvc.Razor/Compilation/CompilationFailedException.cs

```diff
@@ -19,12 +20,19 @@ public class CompilationFailedException : Exception, ICompilationException
         /// details of the compilation failure.</param>
         public CompilationFailedException(
                 [NotNull] ICompilationFailure compilationFailure)
-            : base(Resources.FormatCompilationFailed(compilationFailure.SourceFilePath))
+            : base(FormatMessage(compilationFailure))
         {
             CompilationFailures = new[] { compilationFailure };
         }
 
         /// <inheritdoc />
         public IEnumerable<ICompilationFailure> CompilationFailures { get; }
+
+        private static string FormatMessage(ICompilationFailure compilationFailure)
```

Comment:
We changed Diagnostics \ Helios to not print compilation messages. So we can do better with showing exception messages in the debugger now.